static_src/components/users_invite.jsx

```diff
@@ -0,0 +1,73 @@
+
+/**
+ * Renders a form that allows org users to invite new users
+ * to cloud.gov
+ */
+
+import React from 'react';
+
+import { Form, FormText } from './form';
```

Comment:
I think imports should be organized by:
- external libs (like react)
- space
- Capitalized names in alpha order
- capitalized names in alpha order

So it would be more like this:
```
import React from 'react';

import Action from './action.jsx';
import { Form, FormText } from './form';
import FormStore from '../stores/form_store';
import PanelDocumentation from './panel_documentation.jsx';
import userActions from '../actions/user_actions';
import { validateString } from '../util/validators';
```